src/overlays/actors/ovl_Item_Etcetera/z_item_etcetera.h

```diff
@@ -5,12 +5,19 @@
 
 struct ItemEtcetera;
 
+#define ITEMETCETERA_GET_FF(thisx) ((thisx)->params & 0xFF)
+#define ITEMETCETERA_GET_TREASURE(thisx) (((thisx)->params >> 8) & 0x1F)
+
 typedef void (*ItemEtceteraActionFunc)(struct ItemEtcetera*, GlobalContext*);
 
 typedef struct ItemEtcetera {
-    /* 0x0000 */ Actor actor;
-    /* 0x0144 */ char unk_144[0x10];
-    /* 0x0154 */ ItemEtceteraActionFunc actionFunc;
+    /* 0x000 */ Actor actor;
+    /* 0x144 */ ItemEtceteraActionFunc funcSetup;
```

Comment:
I don't really like this name since it's not clear in the code what the function actually is. It's really the actionfunction to use after the object is loaded. Again, OoT's `futureActionFunc` is quite a good name.